perliedman/elevation-service
# Ignore

Upgrading the base image to `node:lts` works with no other changes required.

Testing:
1. Build Docker image (see repo Readme)
2. Start Docker image (see repo Readme)
3. Exec into docker image `docker exec -it elevation-service /bin/bash`
4. Run tests `npm test`
